.github/ISSUE_TEMPLATE/02-question.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Questions
+about: questions or inquiries about the project
+title: "question: affected/package: "
+labels: question
```

Comment:
I suddenly realized that the [default question label on GitHub](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels#about-labels) actually refers to the issue being in an "in question" state, rather than indicating that the issue is a question raised by a user.

Reply:
"in question": the matter in question: at issue, being discussed, under discussion, under consideration, on the agenda, for debate, to be discussed, to be decided.

Reply:
`question | Indicates that an issue, pull request, or discussion needs more information`

这个描述很宽泛,已经将用户提出的问题这个含义包含在内了